pom.xml

```diff
@@ -24,17 +24,21 @@
   <artifactId>environment-driver</artifactId>
   <version>1.0.0-SNAPSHOT</version>
   <properties>
-    <surefire-plugin.version>2.22.1</surefire-plugin.version>
-    <maven.compiler.target>11</maven.compiler.target>
-    <quarkus.platform.version>1.10.5.Final</quarkus.platform.version>
+
+
+
```

Comment:
```<quarkus.package.type>uber-jar</quarkus.package.type> ```
Please add this so that Quarkus produces uber jar. It's now off by default.

https://quarkus.io/guides/maven-tooling#uber-jar-maven

Reply:
Oh I noticed it's in its separate section. Could you merge it with the properties on top of the pom.xml?